src/reggae/dub/interop/dublib.d

```diff
@@ -416,7 +396,35 @@ class InfoGenerator: ProjectGenerator {
                 return true;
             }
         }
+        static bool isLinkerDFlag_LDC(string arg) {
+            // extra addition for ldmd2
+            if (arg == "-m32mscoff")
+                return true;
+
+            if (arg.length > 2 && arg.startsWith("--"))
+                arg = arg[1 .. $]; // normalize to 1 leading hyphen
+
+            switch (arg) {
+                case "-g", "-gc", "-m32", "-m64", "-shared", "-lib",
+                     "-betterC", "-disable-linker-strip-dead", "-static":
+                    return true;
+                default:
+                    return arg.startsWith("-L")
+                        || arg.startsWith("-Xcc=")
+                        || arg.startsWith("-defaultlib=")
+                        || arg.startsWith("-platformlib=")
+                        || arg.startsWith("-flto")
+                        || arg.startsWith("-fsanitize=")
+                        || arg.startsWith("-link-")
+                        || arg.startsWith("-linker=")
+                        || arg.startsWith("-march=")
+                        || arg.startsWith("-mscrtlib=")
+                        || arg.startsWith("-mtriple=");
+            }
+        }
```

Comment:
Could this function be deleted now that these PRs are merged:
* https://github.com/dlang/dub/pull/2082
* https://github.com/dlang/dub/pull/2083
* ?